src/client/pages/PasscodeEmailSent.tsx

```diff
@@ -153,8 +172,28 @@ export const PasscodeEmailSent = ({
 				disableOnSubmit
 				shortRequestId={shortRequestId}
 			>
-				<PasscodeInput passcode={passcode} fieldErrors={fieldErrors} />
+				<PasscodeInput
+					passcode={passcode}
+					fieldErrors={fieldErrors}
+					label={text.passcodeInputLabel}
+				/>
 			</MainForm>
+			{showSignInWithPasswordOption && (
+				// <LinkButton
+				// 	cssOverrides={secondaryButtonStyles()}
+				// 	priority="tertiary"
+				// 	href={`${buildUrl('/signin/password')}${queryString}`}
+				// >
+				// 	Sign in with password instead
+				// </LinkButton>
+				<>
```

Comment:
Still need this fragment?

Reply:
Nope! And I don't need the comment above anymore either.